packages/jellycommands/src/JellyCommands/commands/CommandManager.ts

```diff
@@ -24,19 +25,35 @@ export class CommandManager {
     }
 
     async respond(interaction: Interaction): Promise<void> {
-        if (!(interaction.isCommand() || interaction.isContextMenu())) return;
+        if (
+            !(
+                interaction.isCommand() ||
+                interaction.isContextMenu() ||
+                interaction.isAutocomplete()
+            )
+        )
+            return;
```

Comment:
Might be good to put the condition in a variable first to avoid the formatting mess